README.md

```diff
@@ -137,14 +164,18 @@ easyMDE.value('New input for **EasyMDE**');
 - **toolbar**: If set to `false`, hide the toolbar. Defaults to the [array of icons](#toolbar-icons).
 - **toolbarTips**: If set to `false`, disable toolbar button tips. Defaults to `true`.
 
+
+### Options example
+
+Most options demonstrate the non-default behavior:
+
 ```JavaScript
-// Most options demonstrate the non-default behavior
 var editor = new EasyMDE({
 	autofocus: true,
 	autosave: {
 		enabled: true,
 		uniqueId: "MyUniqueID",
-		delay: 1000,
```

Comment:
A personal preference, but I always end multi-line objects with a trailing comma.

Reply:
Ok, so maybe we can add [`comma-dangle` rule](https://eslint.org/docs/rules/comma-dangle) to `.eslintrc`? I looked at this file before suggesting this.

Reply:
It's probably best to follow our `.eslintrc` in the example code. If it's not in `.eslintrc` then your change was good.

Reply:
The `.eslintrc` doesn't actually specify what to use. ;)

We could add this rule:

```yml
"comma-dangle": [
    "error",
    "always"
]
```

There are other options such as `always-multiline` or `only-multiline` take a look to [the rule](https://eslint.org/docs/rules/comma-dangle) and choose one according to your preferences, I can make a new PR for this. :)

Reply:
My preference is `always-multiline` :)

Reply:
Ok.

If you want to merge #55, maybe do it before, because this will introduce a lot of conflicts. ;)